server/src/controllers/Events/resolver.ts

```diff
@@ -822,18 +796,30 @@ ${unsubscribeOptions}`,
       },
     });
     await deleteEventReminders(id);
-
     const notCanceledRsvps = event.event_users;
 
-    if (notCanceledRsvps.length) {
+    if (!notCanceledRsvps) return null;
```

Comment:
I don't like the current logic, I think this won't cancel the event if there is no members in the events

Reply:
Should we do something if there are no Members in Events

Reply:
Nope event without members can be canceled, false alarm 👍

Reply:
The fact you could delete an entire mutation and everything passed suggests we have a bit of a testing gap!

Reply:
whelp, I have made it fail with `!notCanceledRsvps.length`, I am openning an issue for that 🤣